viewlet/tests/test_viewlet.py

```diff
@@ -341,15 +338,13 @@ def test_jinja_template_from_dir(self):
         )
         html = template.render({"request": {"user": "nicolas cage"}})
         self.assertTrue(isinstance(html, str))
-        self.assertEqual(html, "nice to see you nicolas cage!")
+        self.assertEqual(html, "nice to see you nicolas cage!\n")
 
-    @skipIf(django.VERSION < (1, 8), "Django < 1.8")
     def test_jinja_template_from_dir_warning(self):
         settings.VIEWLET_TEMPLATE_ENGINE = "jinja2"
         template = self.get_jinja_template("{% viewlet 'hello_from_dir' %}")
-        func = self.assertRaisesRegex if PY3 else self.assertRaisesRegexp
-        with func(DeprecationWarning, "^VIEWLET_TEMPLATE_ENGINE .*"):
-            template.render()
+        func = self.assertRaisesRegex
+        template.render()
```

Comment:
We should keep `with self.assertRaisesRegex` no?

Reply:
It doesn't raise it anymore, but since it's all this test tests for, maybe we should rather remove the test? I'm not sure why it's not deprecated anymore, that's not how those things usually work.

Reply:
It used to be in `get_loader`. Check the diff. But if we deprecated it beforehand, then maybe it should be removed? It doesn't seem to be used anymore.